### PR TITLE
test: use separate ports in tests

### PR DESCRIPTION
**Pull request by TimoGlastra**
Signed-off-by: Timo Glastra <timo@animo.id>

### CODE REVIEW

File: packages/rest/jest.config.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ const config: Config.InitialOptions = {
   ...base,
   name: '@aries-framework/rest',
   displayName: '@aries-framework/rest',
-  testTimeout: 120000,
+  testTimeout: 12000,
```
Comment (janrtvld) [inline packages/rest/jest.config.ts:9]: Is this not used from the base config?

**Review by janrtvld — APPROVED** [review]
Why the random port changes?

File: packages/rest/jest.config.ts (hunk 1 above)
Comment (TimoGlastra) [reply packages/rest/jest.config.ts:9]: Ah no this should be changed back

Comment (TimoGlastra) [inline packages/rest/jest.config.ts:9]:
```suggestion
  testTimeout: 120000,
```

**Pull request merged**